Refresh Nixpkgs' skip-tags patch for OpenBSD 8.0.
LIBC_TAGS=no omits the unused ctags database from the libc build.
https://marc.info/?l=openbsd-tech&m=171575286706032&q=raw

--- a/lib/libc/Makefile
+++ b/lib/libc/Makefile
@@ -29,7 +29,14 @@
 CFLAGS+=-I${LIBCSRCDIR}
 .include "${LIBCSRCDIR}/Makefile.inc"
 
+all: ${SRCS}
+
+LIBC_TAGS ?= yes
+.if ${LIBC_TAGS} == "yes"
 all: tags
+beforeinstall: install_tags
+.endif
+
 tags: ${SRCS}
 	ctags -w ${.ALLSRC:M*.c}
 	egrep "^SYSENTRY(.*)|^ENTRY(.*)|^FUNC(.*)|^SYSCALL(.*)" \
@@ -37,7 +44,8 @@
 	    sed "s;\([^:]*\):\([^(]*\)(\([^, )]*\)\(.*\);\3	\1	/^\2(\3\4$$/;" \
 	    >> tags; sort -o tags tags
 
-beforeinstall:
+.PHONY: install_tags
+install_tags:
 	${INSTALL} ${INSTALL_COPY} -o ${BINOWN} -g ${BINGRP} -m 444 tags \
 		${DESTDIR}/var/db/lib${LIB}.tags
 
